Compiling source
	intg_tests/dataproviders:0.0.0
WARNING [tests/new-data-provider-tests.bal:(121:9,121:21)] unused variable 'a'
WARNING [tests/new-data-provider-tests.bal:(153:9,153:21)] unused variable 'a'

Running Tests with Coverage

	dataproviders
		[pass] testDividingValuesNegative#0
		[pass] testDividingValuesNegative#1
		[pass] testDividingValuesNegative#3
		[pass] testDividingValuesNegative#4
		[pass] testExecutionOfDataValueFailing

		[fail] testDividingValuesNegative#2:

		    [fail data provider for the function testDividingValuesNegative]
			error {ballerina/test:0}ExecutionError ("error("{ballerina}DivisionByZero",message=" / by zero")
				callableName: testDividingValuesNegative moduleName: intg_tests.dataproviders$test.0.tests.new-data-provider-tests fileName: tests/new-data-provider-tests.bal lineNumber: 191
				callableName: testDividingValuesNegative$lambda14$ moduleName: intg_tests.dataproviders$test.0.tests.test_execute-generated_*****lineNumber: 17
			",functionName="testDividingValuesNegative")
				callableName: executeTestFunction moduleName: ballerina.test.0 fileName: execute.bal lineNumber: 477
				callableName: executeDataDrivenTest moduleName: ballerina.test.0 fileName: execute.bal lineNumber: 252
				callableName: prepareDataDrivenTest moduleName: ballerina.test.0 fileName: execute.bal lineNumber: 241
			


		5 passing
		1 failing
		0 skipped

	dataproviders.module1


		No tests found

Generating Test Report
	data-providers/target/report/test_results.json

error: there are test failures